zeebe/stream-platform/src/main/java/io/camunda/zeebe/stream/impl/AsyncProcessingScheduleServiceActor.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.stream.impl;
+
+import io.camunda.zeebe.scheduler.Actor;
+import io.camunda.zeebe.scheduler.ActorControl;
+import io.camunda.zeebe.scheduler.future.ActorFuture;
+import io.camunda.zeebe.scheduler.future.CompletableActorFuture;
+import java.util.Map;
+
+public final class AsyncProcessingScheduleServiceActor extends Actor {
```

Comment:
:upside_down_face: This can likely be package private, same with most of its methods. It's really implementation details.